spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/ArgumentSanitizer.java

```diff
@@ -204,4 +214,75 @@ public HttpHeaders sanitizeHeaders(HttpHeaders headers) {
         }
         return result;
     }
+	/**
+	 * Will replace sensitive string value in the Map with '*****'
+	 *
+	 * @param input to be sanitized
+	 * @return the sanitized map.
+	 */
+	public Map<String, Object> sanitizeMap(Map<String, Object> input) {
```

Comment:
I like where we landed on the code in https://github.com/spring-cloud/spring-cloud-dataflow/commit/5fd5de9ecd44f6cf1444e1435be4fd76c579e896#diff-172ee20c15c7b085b3c8f9366fa27921509974503640d41e9ba8fde51ac5a048R108 . Can we bring that over here as well?